googleapis-raw/src/api/servicecontrol/v1/mod.rs

```diff
@@ -13,12 +13,14 @@
 // limitations under the License.
 
 pub(crate) use crate::{
+    api::distribution as api_distribution,
```

Comment:
Note: I had to make this change and also rename in the `distribution.rs` generated file so it doesn't conflict.  Not sure if there is a better way to do this and this will have to be done manually every time files are re-generated.

Reply:
No, sadly, that's kind of common. There's a few `type.rs` that creep in every so often. It's one of the reasons why I don't purge the `mod.rs` files unless I absolutely have to.